sdks/python/apache_beam/runners/pipeline_context.py

```diff
@@ -29,6 +29,18 @@
 from apache_beam.transforms import core
 
 
+class Environment(object):
+  def __init__(self, proto):
+    self._proto = proto
```

Comment:
Should we instead extract url from proto and store it?
reference https://github.com/apache/beam/blob/2913cbd10d39d890b36a15dca0fc7000701e600e/model/pipeline/src/main/proto/beam_runner_api.proto#L752

Reply:
Doesn't really provide any value to do that, and this is more future-proof (in case we use something other than/in addition to URLs in the future).